communications/newsletters/newsletter_09_Jan2020.md

```diff
@@ -0,0 +1,132 @@
+
+There are many relevant workshops taking place this month, including two workshops in Cambridge on reproducibility using Binder. Find the details below and register soon!
+
+We thank you for your participation and feedback in 2019, which made it a productive year for the project (read our last newsletter [here](https://tinyletter.com/TuringWay/archive)). We look forward to welcoming your ideas and supporting your contributions to the community moving forward. 💞💞
+
+Get in touch with us through various channels listed below this newsletter, and find more details on other topics below 👇
+
+## Community meetings
+
+***The Turing Way Collaboration Cafe***
+
+Join us at the next Collaboration Cafe this week on 15 January 2020, 19:00 GMT ([see your local time](https://arewemeetingyet.com/London/2020-01-15/19:00/TuringWay-CollaborationCafe)). Find more details on [this HackMD](https://hackmd.io/@KirstieJane/CollabCafe). 
+At this first online community meeting, we would like to facilitate an online workspace for The Turing Way Project while getting to know our community members and discussing ways to support their participation and contributions in 2020.
+If you are not sure what to expect from this meeting, check out a few videos recorded in the previous collaboration cafe events on [our YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA).
+ 
+***Open call for The Turing Way Book Dash***
+
+Deadline has been extended to **19 January 2020** to send your application for the first Book Dash, which is taking place on 20 and 21 February. Fill this [short online form](https://docs.google.com/forms/d/e/1FAIpQLSelc8XvuYfz7uIjS1H_RzaK9JI9w9Pc3IfFT4sethro-_6tlw/viewform) by following a step by step guide shared by Kirstie on [this Twitter thread](https://twitter.com/kirstie_j/status/1210934696248586241).
+It is an in-person collaborative event on “how to data science” resource development, where you will work with others to add to and improve [the Turing Way book](https://github.com/alan-turing-institute/the-turing-way). 
+
+
+This event will start on the evening of 20 February 2020 with an introduction to the project, which will be followed by an official dinner. 
+The complete day of 21 February 2020 will be dedicated to the project content development. 
+Your contributions may include, but are not limited to: science communication - skills in writing, editing and/or exciting interactive communications, great examples and case studies of reproducible research, writing new chapters, editing the current chapters and updating how the book looks online (CSS, logos, layout etc). 
+
+## News from the community
+
+***Boost your research reproducibility with Binder workshops in Cambridge***
+ 
+[The British Antarctic Survey](www.bas.ac.uk) is hosting a half-day workshop on 31 January 2020 from 13:30 to 17:00 on research reproducibility in The Turing Way project. 
```

Comment:
```suggestion
[The British Antarctic Survey](https://www.bas.ac.uk/) is hosting a half-day workshop on 31 January 2020 from 13:30 to 17:00 on research reproducibility in The Turing Way project. 
```

Reply:
The link originally didn't work from me - GitHub interprets it as a relative file without the prefix.